### PR TITLE
Add run_command() logging

### PR DESCRIPTION
**Pull request by Williangalvani**
trying to make the blueos_startup_update logs more readable.

before:
image

after:
image

### CODE REVIEW

File: core/tools/blueos_startup_update/blueos_startup_update.py [hunk 1]
```diff
@@ -186,31 +186,25 @@ def boot_config_filter_conflicting_configuration_at_session(
 
 def load_file(file_name) -> str:
     command = f'cat "{file_name}"'
-    output = run_command(command, False)
-    logger.info(output)
-    return output.stdout
+    return run_command(command, False, log_output=False).stdout
```
Comment (patrickelectric) [inline core/tools/blueos_startup_update/blueos_startup_update.py:189]: I'm not a big fan of removing the log for such functions

Comment (Williangalvani) [reply core/tools/blueos_startup_update/blueos_startup_update.py:189]: sounds good. I wasn's sure about this one...

**Comment by Williangalvani** [comment]
updated. now the output of "read file" is shown in full:

image

**Review by joaoantoniocardoso — APPROVED** [review]
nice!! Great improvement!

**Review by patrickelectric — APPROVED** [review]

**Pull request merged**